src/elixirServer.ts

```diff
@@ -116,6 +116,23 @@ export class ElixirServer {
         this.sendRequest('DEFL', command, resultCb);
     }
 
+    getDocumentation(document: vscode.TextDocument, position: vscode.Position, callback: Function): void {
+        const wordAtPosition = document.getWordRangeAtPosition(position);
+        const word = document.getText(wordAtPosition);
+        if (word.indexOf('\n') >= 0) {
+            console.error('[vscode-elixir] got whole file as word');
```

Comment:
I noticed we have this clause all over the place. Would you mind explaining to me why/where/how this even happens and why it isn't possible to guard this once?